inji-web/src/__tests__/components/Common/AppToaster.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { AppToaster } from '../../../components/Common/AppToaster';
+import { Provider } from 'react-redux';
+import { reduxStore } from '../../../redux/reduxStore'; // Assuming the store is named reduxStore
+
+describe('AppToaster', () => {
```

Comment:
please use the standard text used in other test files, to be consistant for describe as well

Reply:
Changed according to the Standard format